analysis_module/models.py

```diff
@@ -26,3 +34,59 @@ class FeaturesType(models.TextChoices):
 
     class Meta:
         db_table = "event_status_tracker"
+
+
+class FailedCallback(BaseModel):
+    class FailedStatus(models.IntegerChoices):
+        SUCCESS = 0, "Success"
+        RETRY_MAXED_OUT = 1, "Retries maxed out"
+        FAILED = 2, "Failed"
+        RETRYING = 3, "Retrying"
+
+    # This contains the uuid of AnalysisModuleRequest object
+    request_unique_id = models.UUIDField(unique=True)
+    retries_count = models.PositiveIntegerField(default=0)
+    last_retried_at = models.DateTimeField(null=True)
+    status = models.PositiveIntegerField(
+        choices=FailedStatus.choices,
+        default=FailedStatus.RETRYING,
+    )
+
+    class Meta:
+        db_table = "failed_callback_tracker"
+
+    def resend_request(self):
+        if self.retries_count >= 5:
```

Comment:
instead of hardcode, let's pass this as env var.

Reply:
Done.